internal/imagerunner/imagerunner.go

```diff
@@ -31,6 +34,14 @@ func Done(status string) bool {
 
 var ErrResourceNotFound = errors.New("resource not found")
 
+type LogURLNotFoundError struct {
+	RunID string
+}
+
+func (e LogURLNotFoundError) Error() string {
+	return fmt.Sprintf("Could not find log URL for run with ID %s", e.RunID)
```

Comment:
Go's error message convention (can't start with upper):
```suggestion
	return fmt.Sprintf("could not find log URL for run with ID %s", e.RunID)
```